lib/go-atscfg/parentdotconfig_test.go

```diff
@@ -258,8 +258,8 @@ func TestMakeParentDotConfigCapabilities(t *testing.T) {
 
 	lines := strings.Split(txt, "\n")
 
-	if len(lines) != 4 {
-		t.Fatalf("expected 4 lines (comment, ds, dot remap, and empty newline), actual: '%+v' text %v", len(lines), txt)
+	if len(lines) != 5 {
+		t.Fatalf("expected 4 lines (comment, blank, ds, dot remap, and empty newline), actual: '%+v' text %v", len(lines), txt)
```

Comment:
This expectation is changed; "expected <del>4</del><ins>5</ins> lines..."

Reply:
Changed